### DISCUSSION TITLE
redpanda: how to help the docker integration

### DISCUSSION THREAD

**Discussion by emaxerrno** [OADA/server#99] [Q&A]
Hi all, cool project. 

I saw a bunch of TODO's/questions here: https://github.com/OADA/server/blob/master/docker-compose.yml

This is a cool project. How can we help make the docker experience better.

**Comment by aultac** [comment]
Hi Alex,

Thanks for reaching out!  redpanda has been a great switch for us.

The “Alex” on our side @awlayton has been taking point on our docker setup, hopefully he can speak to some ideas to improve the docker experience setting up redpanda.  He’s currently embroiled in trying to migrate us to k8s, which may also bring some interesting thoughts.

Thanks!
Aaron

**Reply by emaxerrno** [reply]
hi Aaron! sounds good. let me know if we can help w/ k8s as well. 
also, if helpful to your community, i'm happy to host your use case on our blog as well. (alex@vectorized.io if you want to move to email) 

in any case, here to help
